libafl/src/events/launcher.rs

```diff
@@ -744,11 +742,16 @@ where
 
         #[cfg(feature = "multi_machine")]
         // Create this after forks, to avoid problems with tokio runtime
-        let (multi_machine_sender_hook, multi_machine_receiver_hook) =
+
+        // # Safety
+        // The `multi_machine_receiver_hook` needs messages to outlive the receiver.
+        // The underlying memory region for incoming messages lives longer than the async thread processing them.
```

Comment:
@rmalmain Are we sure this is the case? Is there any race condition when memory gets cleared up before the async thread kicks in?

Reply:
afaik the messages live long enough. i'll double check this